### DISCUSSION TITLE
Feature suggestion

### DISCUSSION THREAD

**Discussion by MarcFinns** [teslamate-org/teslamate#1479] [Ideas]
Hi, it appears as the only filtering is time based. Therefor it is a bit cumbersome to answer questions like “what did I do since last charge”. A specific filter/button would be very useful. 
My 2c. Thanks for the great work, by the way.

**Comment by adriankumpf** [comment]
Moving this to [💡 Ideas](https://github.com/adriankumpf/teslamate/discussions/categories/ideas)
